src/assets/testimonials.js

```diff
@@ -3,21 +3,21 @@ import logoPlaceholder from './logoPlaceholder.png'
 export const testimonials = [
   {
     review:
-      '"Testimonial Lorem ipsum dolor sit amet consectetur adipisicing elit. Placeat nihil quo nostrum rem. Officia cupiditate repellendus,  saepe rem deleniti voluptatibus illum aliquid magnam non distinctio "',
+      'Flex Coast har genom hela processen varit lyhörda till våra behov och snabba i sin respons och sitt arbete. Jag kan starkt rekommendera att ta hjälp från Flex Coast!',
     name: 'Johnny Cage',
     company: 'CEO of MK',
     logo: logoPlaceholder,
   },
   {
     review:
-      '"Officia cupiditate"',
-    name: 'Bob Kramer',
-    company: 'CEO of Kramer Software',
+      'Från första samtalet med Viktor på Flex Coast så har allt fungerat fläckfritt. Alltid tillgänglig, alltid stenkoll och aldrig rädd för att ge det lilla extra.',
```

Comment:
if you add a language key to `review` you can modify the displayed message dependent on UI language:

```js
review: {
  en: "in english....",
  se: "in swedish...
}
```